### PR TITLE
Feature list bills

### PR DESCRIPTION
**Pull request by at-dungnguyen2**
i have already add new feature list bills , please review and merge.
Thanks & best Regards!

### CODE REVIEW

File: resources/views/bills/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+@section('page-content')
+  <div class="row">
+    <div class="col-md-12 col-sm-12 col-xs-12">
+      <div class="clearfix"></div>
+      <table id="datatable-buttons" class="table table-striped table-bordered">
+        <thead>
+          <tr>
+            <th>#</th>
+            <th>Name</th>
```
Comment (trandaison) [inline resources/views/bills/index.blade.php]: Avoid hard code here (table `th` tags)
@at-sontd => please add [rank ...] prefix on your comment

File: config/common_constant.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
```
Comment (dzungnguyenh) [inline config/common_constant.php]: [rank A] This file name should be common.php. That's enough

**Comment by at-dungnguyen2** [comment]
@at-dungnh i have fixed your comments, please review and merge.
thank you & best regards!

File: resources/views/bills/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+@section('section-title')
+  @lang('common.item_list_bill')
+@stop
+
+
+@section('page-content')
+  <div class="row">
+    <div class="col-md-12 col-sm-12 col-xs-12">
+      <div class="clearfix"></div>
+      <table id="datatable-buttons" class="table table-striped table-bordered">
```
Comment (dzungnguyenh) [inline resources/views/bills/index.blade.php]: [rank A] What does "datatable-buttons" mean?

**Comment by at-dungnguyen2** [comment]
@at-dungnh I have fixed your comments, please review and merge.
Thank you & best regards!

**Comment by dzungnguyenh** [comment]
LGTM. Thanks.

**Pull request merged**